animations/lib/src/misc/physics_card_drag.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/physics.dart';
+
+class PhysicsCardDragDemo extends StatelessWidget {
+  static const String routeName = '/misc/physics_card';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(),
+      body: DraggableCard(
+        child: FlutterLogo(
+          size: 128,
+        ),
+      ),
+    );
+  }
+}
+
+/// A draggable card that moves back to [Alignment.center] when it's
+/// released.
+class DraggableCard extends StatefulWidget {
+  final Widget child;
+  DraggableCard({this.child});
+
+  @override
+  _DraggableCardState createState() => _DraggableCardState();
+}
+
+class _DraggableCardState extends State<DraggableCard>
+    with SingleTickerProviderStateMixin {
+  AnimationController _controller;
+
+  /// The position the user has dragged the card to.
```

Comment:
nit: This isn't quite true, since it could also be the current position of the card snapping back from a drag. Consider something like:

"The current alignment of the DraggableCard within its parent. While the card is being dragged, this value is set in the GestureDetector's callbacks. When the drag ends, an animation updates this value until it returns to Offset.zero."

Reply:
That's a good clarification. I changed this to:

```
  /// The alignment of the card as it is dragged or being animated.
  ///
  /// While the card is being dragged, this value is set to the values from
  /// derived in the GestureDetector onPanUpdate callback. If the animation is
  /// running, this value is set to the value of the [_animation].
```